mozilla/blurts-server
# Add links to company websites for breach resolution

# References: 
Jira: [MNTOR-1504](https://mozilla-hub.atlassian.net/browse/MNTOR-1504)



# Description

Adds a link to the breach resolution when passwords or security questions were involved.

# Screenshot

<img width="411" alt="MNTOR-1504" src="https://user-images.githubusercontent.com/13835474/229155764-f0a9ba34-de5d-4b7d-bafb-e0856e6adeae.png">

# How to test

1. Login with an account or add an email address that was exposed in a breach.
2. Visit `/user/breaches`
3. If the breach includes passwords or security questions there should be a link to the company website in the associated resolution step.

# Checklist (Definition of Done)
